src/notifications.c

```diff
@@ -88,7 +97,17 @@ int HashNotificationCallback(RedisModuleCtx *ctx, int type, const char *event,
   else CHECK_CACHED_EVENT(rename_from)
   else CHECK_CACHED_EVENT(rename_to)
   else CHECK_CACHED_EVENT(loaded)
-//  else CHECK_CACHED_EVENT(json_set)
+  else CHECK_CACHED_EVENT(json_set)

```

Comment:
Do we know this is kosher? I can spot "json_set" (for instance) in a couple of spots in RedisJSON and Rust might not unify the strings.

Reply:
You are right about this one.
@oshadmi, can we unify them on RedisJSON side?

Reply:
@ashtul @rafie It seems that Rust is using the same address for similar string slices (`&str`), so both `"json_set"` literals are actually already being unified.
However, when calling the API `RedisModule_NotifyKeyspaceEvent`, we are constructing a new `CString` from the string slice (to work according to Rust FFI  (Foreign Function Interface)) and passing it to the API, so the notification callback could be receiving a different pointer each time for the `event` parameter and currently string comparison seems to be required.

Reply:
Done!